entry_points.ini

```diff
@@ -41,3 +41,4 @@
 [console_scripts]
 set-image-in-bss = libcsm.bss.set_image:main
```

Comment:
Looking at these 3 commands, I think they need a namespace.

```
bss-set-image
sls-get-xname
sls-get-hostname
```

I think later we can improve this by adding a `click` file, so we can have `sls get hostname` for example. For now, prepending the command with the name of the endpoint it's talking to would be better.

Reply:
Changed